docs/examples/webpack/babel.config.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Babel config
+ *
+ * @type {import('@babel/core').ConfigFunction}
+ */
+module.exports = function (api) {
+  const browserslistEnv = !api.env('test')
+    ? 'javascripts'
+    : 'node'
+
+  return {
+    presets: [
+      ['@babel/preset-env', {
+        browserslistEnv,
+        loose: browserslistEnv !== 'node',
+
+        // Transform ES modules for Node.js
+        modules: browserslistEnv === 'node' ? 'auto' : false
+      }]
+    ]
+  }
```

Comment:
Given that all of the config for '@babel/preset-env' depends on the `browsersListEnv`, I'm wondering if it might be simpler to just return two different objects? Especially as there's a lot of negation going on at the minute which can make it harder to follow.

```suggestion
  let presetEnvConfig

  if (api.env('test')) {
    presetEnvConfig = {
      browsersListEnv: 'node',
      modules: false
    }
  } else {
    presetEnvConfig = {
      browsersListEnv: 'javascripts',
      loose: true
    }
  }

  return {
    presets: [
      ['@babel/preset-env', presetEnvConfig]
    ]
  }
```

Or even:
```suggestion
  if (api.env('test')) {
    return {
      presets: [
        ['@babel/preset-env', {
          browsersListEnv: 'node',
          modules: false
        }]
      ]
    }
  } else {
    return {
      presets: [
        ['@babel/preset-env', {
          browsersListEnv: 'javascripts',
          loose: true
        }]
      ]
    }
  }
```

Reply:
I'm happy at the moment. Would you rather we did?

Reply:
@36degrees I've removed `!== 'node'` to help keep [`examples/webpack/babel.config.js`](https://github.com/alphagov/govuk-frontend/blob/browserslist-support/docs/examples/webpack/babel.config.js) short and sweet

But only because we're going to need to duplicate the config for the `govuk-frontend` package over in:

* https://github.com/alphagov/govuk-frontend/pull/3820